contracts/RiskFund/RiskFund.sol

```diff
@@ -88,23 +89,14 @@ contract RiskFund is Ownable2StepUpgradeable, ExponentialNoError, ReserveHelpers
         emit PoolRegistryUpdated(oldPoolRegistry, _poolRegistry);
     }
 
-    /**
-     * @dev Convertible base asset setter
-     * @param _convertibleBaseAsset Address of the asset.
-     */
-    function setConvertableBaseAsset(address _convertibleBaseAsset) external onlyOwner {
-        require(_convertibleBaseAsset != address(0), "Risk Fund: Asset address invalid");
-        address oldBaseAsset = convertibleBaseAsset;
-        convertibleBaseAsset = _convertibleBaseAsset;
-        emit ConvertableBaseAssetUpdated(oldBaseAsset, _convertibleBaseAsset);
-    }
-
     /**
      * @dev Shortfall contract address setter
      * @param _shortfallContractAddress Address of the auction contract.
      */
     function setShortfallContractAddress(address _shortfallContractAddress) external onlyOwner {
         require(_shortfallContractAddress != address(0), "Risk Fund: Shortfall contract address invalid");
+        require(IShortfall(_shortfallContractAddress).convertibleBaseAsset() != address(0), "Risk Fund: Base asset doesn't match");
```

Comment:
Shouldn't we check this: `IShortfall(_shortfallContractAddress).convertibleBaseAsset() == convertibleBaseAsset`?

Reply:
Yes right. I will fix this